examples/tv-app/android/java/AppImpl.cpp

```diff
@@ -462,7 +416,19 @@ EndpointId AddContentApp(const char * szVendorName, uint16_t vendorId, const cha
     ContentAppImpl * app =
         new ContentAppImpl(szVendorName, vendorId, szApplicationName, productId, szApplicationVersion, "20202021", manager);
     ChipLogProgress(DeviceLayer, "AppImpl: AddContentApp vendorId=%d applicationName=%s ", vendorId, szApplicationName);
-    return gFactory.AddContentApp(app, manager);
+    return gFactory.AddContentApp(app);
+#endif // CHIP_DEVICE_CONFIG_APP_PLATFORM_ENABLED
+    return 0;
+}
+
+EndpointId AddContentApp(const char * szVendorName, uint16_t vendorId, const char * szApplicationName, uint16_t productId,
+                         const char * szApplicationVersion, EndpointId endpointId, jobject manager)
+{
+#if CHIP_DEVICE_CONFIG_APP_PLATFORM_ENABLED
+    ContentAppImpl * app =
+        new ContentAppImpl(szVendorName, vendorId, szApplicationName, productId, szApplicationVersion, "20202021", manager);
+    ChipLogProgress(DeviceLayer, "AppImpl: AddContentApp vendorId=%d applicationName=%s ", vendorId, szApplicationName);
+    return gFactory.AddContentApp(app, endpointId);
 #endif // CHIP_DEVICE_CONFIG_APP_PLATFORM_ENABLED
     return 0;
```

Comment:
is 0 considered failure? Should it be some constant named kInvalidEndpointId for clarity?

Reply:
I can change that.